.github/workflows/update-links.yml

```diff
@@ -31,15 +32,18 @@ jobs:
         env:
           CF_ACCESS_KEY_ID: ${{ secrets.CF_ACCESS_KEY_ID }}
           CF_SECRET_ACCESS_KEY: ${{ secrets.CF_SECRET_ACCESS_KEY }}
-
-      - name: Open PR with updates
-        uses: peter-evans/create-pull-request@v5
+      
+      - name: Commit Changes
+        uses: stefanzweifel/git-auto-commit-action@8756aa072ef5b4a080af5dc8fef36c5d586e521d # v5.0.0
         with:
-          commit-message: Update Redirect links
-          title: "chore: update redirect links"
-          body: |
-            This PR updates the links for url redirects.
-          branch: update-redirect-links
-          delete-branch: true
-          base: main
+          branch: main
+          commit_options: '--no-verify --no-signoff'
+          commit_message: 'chore: update redirect links'
+          file_pattern: 'src/constants/redirectLinks.json'
```

Comment:
Is a file patrtern really needed? It will by default commit any changed file, which in case happens here "Update Redirect Links" 🤔 

But fine keeping it

Reply:
I added this since I generally prefer more verbosity, can definitely remove if y'all think it'd look better though, same with `branch: main`

Reply:
Right, it's just if we change the path of this file and forget to update this line we have a problem.